### PR TITLE
React "Installation" page translated into hindi.

### PR DESCRIPTION
**Pull request by khizarq313**
Hindi translation of "Installation" page of React Documentation.

### CODE REVIEW

File: src/content/learn/installation.md [hunk 1]
```diff
@@ -1,25 +1,25 @@
 ---
-title: Installation
+title: इंस्टॉलेशन
 ---
 
 <Intro>
 
-React has been designed from the start for gradual adoption. You can use as little or as much React as you need. Whether you want to get a taste of React, add some interactivity to an HTML page, or start a complex React-powered app, this section will help you get started.
+React को शुरु से ही धीरे-धीरे अपनाने के लिए डिज़ाइन किया गया है। तुम जितना चाहो, उतना React का यूज़ कर सकते हो। चाहे आप React की एक झलक पाना चाहते हो, HTML पेज में कुछ इंटरएक्टिविटी जोड़ना चाहते हो, या एक काम्प्लेक्स React-पॉवर्ड ऐप शुरू करना चाहते हो, यह सेक्शन आपकी शुरुआत में मदद करेगा।
```
Comment (arshadkazmi42) [inline src/content/learn/installation.md:7]:
```suggestion
React को शुरु से ही धीरे-धीरे अपनाने के लिए डिज़ाइन किया गया है। आप जितना चाहें, उतना React का यूज़ कर सकते हो। चाहे आप React की एक झलक पाना चाहते हो, HTML पेज में कुछ इंटरएक्टिविटी ऐड करना चाहते हो, या एक काम्प्लेक्स React-पॉवर्ड ऐप शुरू करना चाहते हो, यह सेक्शन आपकी शुरुआत में मदद करेगा।
```

File: src/content/learn/installation.md [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
-* [How to install React Developer Tools](/learn/react-developer-tools)
+* [एक नया React प्रोजेक्ट कैसे शुरू करें](/learn/start-a-new-react-project)
+* [एक मौजूदा प्रोजेक्ट में React कैसे ऐड करें](/learn/add-react-to-an-existing-project)
+* [अपने एडिटर को कैसे सेटअप करें](/learn/editor-setup)
+* [React डेवलपर टूल्स को कैसे इनस्टॉल करें](/learn/react-developer-tools)
 
 </YouWillLearn>
 
-## Try React {/*try-react*/}
+## रिएक्ट को आजमाएं {/*try-react*/}
```
Comment (arshadkazmi42) [inline src/content/learn/installation.md:20]:
```suggestion
## React को आजमाएं {/*try-react*/}
```

File: src/content/learn/installation.md [hunk 3]
```diff
@@ -35,23 +35,22 @@ export default function App() {
 
 </Sandpack>
 
-You can edit it directly or open it in a new tab by pressing the "Fork" button in the upper right corner.
+आप इसे डायरेक्ट एडिट कर सकते हैं या उपरी राईट कोर्नर में 'Fork' बटन दबाकर एक नए टैब में ओपन कर सकते हैं।
 
-Most pages in the React documentation contain sandboxes like this. Outside of the React documentation, there are many online sandboxes that support React: for example, [CodeSandbox](https://codesandbox.io/s/new), [StackBlitz](https://stackblitz.com/fork/react), or [CodePen.](https://codepen.io/pen?&editors=0010&layout=left&prefill_data_id=3f4569d1-1b11-4bce-bd46-89090eed5ddb)
+React डॉक्युमेंटेशन में बहुत से पेज में इस तरह के सैंडबॉक्स शामिल हैं। React डॉक्युमेंटेशन के बाहर, बहुत सारे ऑनलाइन सैंडबॉक्स हैं जो React को सपोर्ट करते हैं: उदाहरण के लिए, [CodeSandbox](https://codesandbox.io/s/new), [StackBlitz](https://stackblitz.com/fork/react) , या [CodePen.](https://codepen.io/pen?&editors=0010&layout=left&prefill_data_id=3f4569d1-1b11-4bce-bd46-89090eed5ddb)
```
Comment (arshadkazmi42) [inline src/content/learn/installation.md:40]:
```suggestion
React डॉक्युमेंटेशन में बहुत से पेज में इस तरह के सैंडबॉक्स शामिल हैं। React डॉक्युमेंटेशन के बाहर, बहुत सारे ऑनलाइन सैंडबॉक्स हैं जो React को सपोर्ट करते हैं: उदाहरण के लिए, [CodeSandbox](https://codesandbox.io/s/new), [StackBlitz](https://stackblitz.com/fork/react) , या [CodePen।](https://codepen.io/pen?&editors=0010&layout=left&prefill_data_id=3f4569d1-1b11-4bce-bd46-89090eed5ddb)
```

**Review by arshadkazmi42 — APPROVED** [review]
Awesome work.

I have added a couple of suggestions, once those are fixed this is good to merge.

**Comment by khizarq313** [comment]
I made all the changes as you suggested!
Thank you for the review!!

**Review by arshadkazmi42 — APPROVED** [review]
Looks good. Thanks

**Pull request merged**